js/src/components/adaptive-form/adaptive-form-context.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+/**
+ * @typedef {Object} InputProps
+ * @property {*} value Form value.
+ * @property {boolean} checked Form value converted to boolean.
+ * @property {*} selected Form value.
+ * @property {(value: Event<HTMLInputElement> | *) => void} onChange Function to handle onChange event.
+ * @property {() => void} onBlur Function to handle onBlur event.
+ * @property {'has-error' | undefined} className 'has-error' if the form value is invalid and marked as touched. `undefined` otherwise.
+ * @property {string | undefined | null} help The corresponding value in form `errors` if the form value is marked as touched. `null` otherwise.
```

Comment:
❓ 

So In which cases it is `undefined`?

Reply:
When an input field is marked as touched and has no errors, the value of `help` will be `undefined`, which come from the corresponding value in form `errors`.
Ref: https://github.com/woocommerce/woocommerce/blob/6.9.0/packages/js/components/src/form/form.tsx#L258

Reply:
> Maybe worth it to at that case in the property docs

I tried this one but looks a bit redundant:
```
The corresponding value in form `errors` if the form value is marked as touched, so whether it's a string or `undefined` depends on the corresponding value in the current form `errors`. `null` otherwise.
```
Do you have any suggestions?